NetworkProvider.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  NetworkProvider.swift
+//  moviesTime
+//
+//  Created by Jose Eduardo on 30/07/21.
+//
+
+import Foundation
+import Alamofire
+
+final class NetworkProvider{
+    
+    static let shared = NetworkProvider()
```

Comment:
if you are going to make a singleton, there must be a private `init` inside the class